when I build jsc, got a compile Error in icu/common/brkiter.cpp

[-] PROCESSING BUCK FILES...FINISHED 0.5s [100%]
[+] DOWNLOADING... (0.00 B/S, TOTAL: 0.00 B, 0 Artifacts)
[+] BUILDING...3.2s [27%] (265/1921 JOBS, 53 UPDATED, 2.8% CACHE MISS)
 |=> //icu:common#android-armv7,shared...  2.1s (running c++ link[2.1s])
 |=> //jsc:llint_desired_offsets...  2.7s (running genrule[2.7s])
 |=> //jsc:wtf#android-x86,compile-pic-WTF_wtf_StringPrintStream.cpp.o...  0.3s (running c++ piped_preprocess_and_compile[0.3s])
 |=> //jsc:wtf#android-x86,compile-pic-WTF_wtf_FilePrintStream.cpp.o...  0.9s (running c++ piped_preprocess_and_compile[0.9s])
 |=> //jsc:wtf#android-x86,compile-pic-WTF_wtf_FunctionDispatcher.cpp.o...  0.4s (running c++ piped_preprocess_and_compile[0.4s])
 |=> //jsc:wtfassertions#android-x86,compile-pic-WTF_wtf_Assertions.cpp.o...  1.1s (running c++ piped_preprocess_and_compile[1.1s])
 |=> //jsc:wtf#android-x86,compile-pic-WTF_wtf_GregorianDateTime.cpp.o...  0.3s (running c++ piped_preprocess_and_compile[0.3s])
 |=> //jsc:wtf#android-x86,compile-pic-WTF_wtf_dtoa_bignum.cc.o...  0.3s (running c++ piped_preprocess_and_compile[0.3s])
 |=> //jsc:wtf#android-x86,compile-pic-WTF_wtf_StackStats.cpp.o...  0.3s (running c++ piped_preprocess_and_compile[0.3s])
 |=> //icu:common#android-x86,compile-pic-common_unistr.cpp.o...  1.3s (running c++ piped_preprocess_and_compile[1.3s])
icu/common/brkiter.cpp:426: error: undefined reference to 'icu_56::FilteredBreakIteratorBuilder::createInstance(icu_56::Locale const&, UErrorCode&)'
icu/common/uresbund.cpp:1906: error: undefined reference to 'ures_getAllArrayItems'
icu/common/uresbund.cpp:1908: error: undefined reference to 'ures_getAllTableItems'
icu/common/uresbund.cpp:1984: error: undefined reference to 'icu_56::ResourceDataValue::~ResourceDataValue()'
icu/common/uresbund.cpp:1985: error: undefined reference to 'vtable for icu_56::ResourceDataValue'
/Users/mxw/Programe/ndk/toolchains/arm-linux-androideabi-4.8/prebuilt/darwin-x86_64/bin/../lib/gcc/arm-linux-androideabi/4.8/../../../../arm-linux-androideabi/bin/ld: the vtable symbol may be undefined because the class is missing its key function
collect2: error: ld returned 1 exit status

@lexs I changed the fetch_sources.sh as your commit, but also got the error

icu/common/brkiter.cpp:426: error: undefined reference to 'icu_56::FilteredBreakIteratorBuilder::createInstance(icu_56::Locale const&, UErrorCode&)'
collect2: error: ld returned 1 exit status

--- a/fetch_sources.sh
+++ b/fetch_sources.sh
@@ -7,7 +7,7 @@ set -e
 cd "$(dirname "$0")"
 
 echo "Downloading ICU"
-curl -o icu4c.tar.gz https://android.googlesource.com/platform/external/icu/+archive/master/icu4c/source.tar.gz
+curl -o icu4c.tar.gz https://android.googlesource.com/platform/external/icu/+archive/e25a54101b72d27b345934e1574aa314c1899969/icu4c/source.tar.gz
 
 echo "Extracting ICU"
 tar -zxf icu4c.tar.gz -C icu

